augur/merge.py

```diff
@@ -0,0 +1,296 @@
+"""
+Merge two or more metadata tables into one.
+
+Tables must be given unique names to identify them in the output and are
+merged in the order given.
+
+Rows are joined by id (e.g. "strain" or "name" or other
+--metadata-id-columns), and ids must be unique within an input table (i.e.
+tables cannot contain duplicate ids).  All rows are output, even if they
+appear in only a single table.
```

Comment:
Is it worth adding something like: 
```suggestion
appear in only a single table. (This is effectively a FULL OUTER JOIN
in SQL terms.)
```

Reply:
Sure. Added something similar in repush.